pyangbind/plugin/pybind.py

```diff
@@ -38,6 +39,17 @@
 from pyang import statements
 from pyang import util
 
+# Temporrary kludge for Python3 support
+try:
+  long
+except NameError:
+  long = int
```

Comment:
I don't think we can workaround this one, but please consider using this pattern:

```python
if sys.version_info > (3,):
  long = int
```

This makes it more explicit what we're checking for here.